libmamba/src/api/channel_loader.cpp

```diff
@@ -86,6 +86,63 @@ namespace mamba
         }
     }
 
+    std::string get_system_certificate()
+    {
+        std::string res = "";
+        if (on_linux)
+        {
+            std::array<std::string, 6> cert_locations{
```

Comment:
should we make this an array of fs::path?

Reply:
Indeed, that would be an improvement